src/web/lib/components/Modal/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import iconClose from "../../../../images/close-16.svg";
+import "./index.scss";
+
+export const SavedThemeSelector = ({ toggleModal, children }) => {
```

Comment:
Forgive my endless ignorance, but is "SavedThemeSelector" legit here (and default export below on L26)? Or should this be something like "Modal" and this was just copy-pasta?

Reply:
Oh yeah, that should be Modal...total copypasta